src/libasr/pass/replace_symbolic.cpp

```diff
@@ -49,263 +49,20 @@ class ReplaceSymbolicVisitor : public PassUtils::PassVisitor<ReplaceSymbolicVisi
     std::set<ASR::symbol_t*> symbolic_vars_to_omit;
     SymEngine_Stack symengine_stack;
 
-    /********************************** Utils *********************************/
-    #define BASIC_CONST(SYM, name)                                              \
```

Comment:
Some or maybe all the changes has to be reverted in this file : https://github.com/lcompilers/lpython/pull/2433